utilities/db-migration/migrate/migrate_db.go

```diff
@@ -61,6 +63,16 @@ func Migrate(opType string, migrationPath string) error {
 			return fmt.Errorf("unable to upgrade migration version by 1 level: %v", err)
 		}
 		return nil
+	} else if opType == "migrate_to" {
+		u64, err := strconv.ParseUint(os.Args[2], 10, 32)
+		if err != nil {
+			fmt.Println(err)
```

Comment:
Looks good. One final question: do we want to return the error if ParseUint fails to parse?

Reply:
I think yes, we need to capture all the errors right? as ParseUint function returns error, please let me know if my assumption is wrong

Reply:
Yes. Please return the error